tests/rule-setup.js

```diff
@@ -13,14 +13,15 @@ describe('rules setup is correct', function() {
     const path = join(__dirname, '../lib/rules');
     const files = readdirSync(path);
 
+    // eslint-disable-next-line node/no-deprecated-api
     assert.deepEqual(
       RULE_NAMES,
       files.filter(file => !file.startsWith('.')).map(file => file.replace('.js', ''))
     );
```

Comment:
maybe we should use `chai` instead of the built-in `assert`? 🤔

Reply:
Any reason to choose `chai` over `assert`? Could consider as a follow-up.